ipaserver/plugins/dogtag.py

```diff
@@ -519,14 +455,28 @@ def parse_updateCRL_xml(doc):
 register = Registry()
 
 
-class RestClient(Backend):
-    """Simple Dogtag REST client to be subclassed by other backends.
+def support_v2(pki_client):
+    """Fix the PKIClient class if the remote CA is 11.7.0 or higher
+
+       Older versions of PKI either do not support v2 or
+       IPA doesn't configure to accept v2. Fall back to v1.
+    """
+    server_info = pki_client.info_client.get_info()
+    version = pki.util.Version(server_info.version)
+    if version <= pki.util.Version('11.6.0'):
```

Comment:
In my tests, I  used PKI 11.7 with IPA 4.12 on the master and the ca/v2/ endpoint is used but IPA ships ipa-pki-proxy.conf with a redirect of "^/(ca|kra|pki)/rest/" to the AJP connector.
Nothing redirects ca/v2 to the AJP connector.